app/src/js/components/Requests/request.js

```diff
@@ -265,10 +327,21 @@ class RequestOverview extends React.Component {
     const { canEdit } = formPrivileges(this.props.privileges);
     const { roles } = this.props;
     const role = roles ? Object.keys(roles).map(role => roles[role].short_name) : [];
+    const { users } = this.props;
+    const filteredUsers = users.filter(user => user.user_roles.find(role => role.short_name == "uwg_member")
+    )
     let workflowSave;
     if (canWithdraw && canRestore) {
       workflowSave = this.renderWorkflowSave(record);
     }
+    let assignUWGReviewers;
+    let canAssignUWG = false;
+    if (role.includes('admin') || role.includes('manager') || role.includes('staff')) {
```

Comment:
In general, we've tried to move away from assigning permissions based on role and instead focused on assigning permissions based on the privileges. I'll look more into how this might map to privileges when validating